printable-recipe-generator/src/main.ts

```diff
@@ -3,8 +3,63 @@ import * as process from 'node:process';
 import path from 'path';
 import type { Data } from 'ejs';
 import { render as renderTemplate } from 'ejs';
+import fetch from 'node-fetch';
+
+//load Contributors
+async function getChefs(): Promise<unknown> {
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call -- response is expected to be JSON, safe to assign
```

Comment:
This line should not be needed, assigning `resp` from `fetch` should not cause an error